test.py

```diff
@@ -37,7 +39,7 @@
     MODES = [args.mode] if args.mode else seastar_cmake.SUPPORTED_MODES
 
     def run_tests(mode):
-        BUILD_PATH = seastar_cmake.BUILD_PATHS[mode]
+        BUILD_PATH = seastar_cmake.build_path(mode, args.build_root)
```

Comment:
test.py gets the same treatment as configure.py.

These are the only two files which call `build_path`.